src/commands.rs

```diff
@@ -562,28 +562,37 @@ struct SplitArgs {
 /// when calculating the set of revisions to rebase. That set includes the
 /// specified revision and all ancestors that are not also ancestors
 /// of the destination. It also includes all descendants of those commits. For
-/// example, `jj rebase -b B -d D` or `jj rebase -b C -d D`  would transform
-/// your history like this:
+/// example, `jj rebase -b C -d F` or `jj rebase -b E -d F`  would transform
+/// your history like this (letters followed by an apostrophe are post-rebase
+/// versions):
 ///
-/// D          B'
-/// |          |
-/// | C        D
-/// | |   =>   |
-/// | B        | C'
-/// |/         |/
-/// A          A
+/// F           D'
+/// |           |
+/// | D         C'
+/// | |         |
+/// | C         | E'
+/// | |    =>   |/
+/// | | E       B'
+/// | |/        |
+/// | B         F
+/// |/          |
+/// A           A
```

Comment:
Cool, looks like `-b` was what I wanted the whole time! It looks like `jj rebase -b X -d Y` is equivalent to `jj rebase -s 'roots(X..Y)' -d Y`? If so, maybe that's worth including at some point if folks find it helpful.

Reply:
Yes, good point. I've updated the text. See if this is readable.